<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora de indice de masa corporal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="pesoIn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="alturaIn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="calcularBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IMC</string>
    </property>
   </widget>
   <widget class="QLabel" name="pesoLb">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Peso</string>
    </property>
   </widget>
   <widget class="QLabel" name="alturaLb">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Altura</string>
    </property>
   </widget>
   <widget class="QLabel" name="interpretacion">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Interpretaci√≥n</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imcOut">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
